### PR TITLE
Pods are no longer miniguns

### LINKED ISSUE #8934: Pods are miniguns
**Problem Description**:
You can shoot an entire pods charge away in an instant
Edit: After testing it at round end its 4 shots per burst but queues up so many clicks you generally click a few hundred times ending your pods charge after a few standard Cooldowns depending on the pod's battery

**What did you expect to happen**: 
For you to not be able to fire like 20 times in an instant

**What happened instead**:
PEW PEW PEW PEW PEW PEW PEW

**Why is this bad/What are the consequences:**
You could literally insta kill someone with a pod

**Steps to reproduce the problem**:
Setup a Fire pod weapons macro on byond.
Set up a macro using an external program that activates your fire pod weapons macro at superhuman speeds.
Blast your enemies away.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Works with disablers and lasers, not sure about mining weapons

### PR DESCRIPTION
**Pull request by alexkar598**
So this used to wait 2 seconds to apply in the cooldown,in that period you could fire the verb again i think,~~not sure~~YAY IM SMART IT WORKS
:cl:alexkar598
bugfix:Pods can no longer fire lasers at the speed of light.Pun intended.
/:cl:

Fixes #8934

### CODE REVIEW

**Comment by Aurorablade** [comment]
the more they fire..the more battery they use...

**Comment by alexkar598** [comment]
gotta wait those 1.5 seconds

File: code/modules/spacepods/equipment.dm [hunk 1]
```diff
@@ -40,8 +41,6 @@
 			playsound(src, fire_sound, 50, 1)
 			projone.dumbfire(my_atom.dir)
 			projtwo.dumbfire(my_atom.dir)
-		sleep(2)
```
Comment (tigercat2000) [inline code/modules/spacepods/equipment.dm]: This sleep is rather important, it prevents the lasers from riding the same tiles

**Comment by alexkar598** [comment]
Haven't noticed any problem while testing,lasers fired from both side of the front of the pod with the correct cool down,the sleep was useless as you can  fire the verb again regardless of the sleep

**Comment by AffectedArc07** [comment]
Can we get the WTF label

**Comment by alexkar598** [comment]
No. It's a bug and it was fixed

**Comment by AffectedArc07** [comment]
Aww

**Pull request merged**